### ISSUE TITLE
Error retrieving a new deposit address in testnet.blinktrade.com

### ISSUE THREAD

**Issue by reinaldomendes** [blinktrade/BlinkTradeJS#24]
Hi,

I'm trying to use the platform in https://testnet.blinktrade.com 
I'm signed up new account.

When I'm trying to generate new deposit address I'm receiving the error bellow:

`Message from server: Error retrieving a new deposit address. Please, try again - <urlopen error [Errno 111] Connection refused>`

**Comment by cesardeazevedo** [comment]
Yes, this issue remains for a while (#19 #21) i will talk to @pinhopro about it, but for now, i suggest you to create a account on foxbit testnet instead (https://testnet.foxbit.exchange) and create a fiat deposit, and then, logged as foxbit testnet admin (username: foxbit, password: abc12345, and don't forget to click on footer admin link) and approve your own deposit on the admin tab.

**Comment by reinaldomendes** [comment]
@cesardeazevedo

The foxbit testnet https://testnet.foxbit.exchange is not working.

![testnet foxbit com](https://user-images.githubusercontent.com/725300/32091985-6b076d88-bad5-11e7-80cc-ca67dec64794.png)

**Comment by cesardeazevedo** [comment]
It's working fine here.

**Comment by reinaldomendes** [comment]
@cesardeazevedo The problem here is the DNS.

```bash
$  ping api.testnet.blinktrade.com
ping: unknown host api.testnet.blinktrade.com

`

**Comment by cesardeazevedo** [comment]
Hi @reinaldomendes 

Can you login as rodrigo/abc12345 directly on testnet.blinktrade.com ? This is also a testnet account that we use.

The testnet.foxbit hasn't been updated for a long time, and the correct endpoint for testnet is `api_testnet.blinktrade.com`, we will try to update it on testnet.foxbit very soon, but for now, you can just use testnet.blinktrade.com

**Comment by reinaldomendes** [comment]
Thanks @cesardeazevedo.

**Comment by reinaldomendes** [comment]
@cesardeazevedo  
The lib installed by npm '0.0.10' uses the endpoint api.testnet.blinktrade.com istead of api_testnet.blinktrade.com.

I replaced the endpoint in all files and the api works, but you should consider update the code.

The issue #25 also occurs in testnet.

Thanks.

**Comment by cesardeazevedo** [comment]
Yes indeed, i need to update this as well, thanks for reporting.

**Comment by ericrf** [comment]
i got the same problem.

**Comment by ericrf** [comment]
same issue, is there a way to fix that?
